importanize/formatters.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function, unicode_literals
+
+import itertools
+import operator
+
+
+def get_normalized(i):
+    return map(operator.attrgetter('normalized'), i)
+
+
+class Formatter(object):
+    """Parent class for all formatters
+
+    Parameters
+    ----------
+    statement : ImportStatement instance
+
+    """
+    def __init__(self, statement):
+        self.statement = statement
+
+    def init(self):
+        """Rename some statement attributes to simplify code."""
+        self.leafs = self.statement.unique_leafs
+        self.stem = self.statement.stem
+        self.comments = self.statement.comments
+        self.string = self.statement.as_string()
+        self.file_artifacts = self.statement.file_artifacts.get('sep', '\n')
+        self.all_comments = (
+            self.comments + list(itertools.chain(
+                *list(map(operator.attrgetter('comments'), self.leafs))
+            ))
+        )
+
+
+class IndentWithTabsFormatter(Formatter):
+    """Default formatter used to organize long imports
+
+    Imports are added one by line preceded by 4 spaces, here's a sample output:
+
+    from other.package.subpackage.module.submodule import (
+        CONSTANT,
+        Klass,
+        bar,
+        foo,
+        rainbows,
+    )
+    """
+
+    def format(self):
+        self.init()
```

Comment:
can this use some other API? I dont like that `init` will be called every time `format` is called.